src/biogeochem/DUSTMod.F90

```diff
@@ -356,86 +567,170 @@ subroutine DustEmission (bounds, &
+         ! use tmp1 from DUSTini for Iversen and White I&W (1982) (~75 um is optimal); use tmp2 for S&L (2000) (~80 um is optimal)
+         !#######################################################################################################
 
-            ! the following comes from subr. frc_thr_rgh_fct_get
-            ! purpose: compute factor by which surface roughness increases threshold
-            !          friction velocity (currently a constant)
+         tmp2 = 1.0_r8*sqrt(0.0123_r8 * (dns_slt*grav*130.0e-6_r8 + 1.65e-4_r8/130.0e-6_r8)) ! calculate S&L (2000) scheme here for threshold; gamma = 1.65e-4 following S&L00, D_p = 127 um ~ 130 um following Leung et al. (2022). As this is a global constant, this line can be put outside the loop to save computational power.
+         wnd_frc_thr_dry(p) = tmp2 / sqrt(forc_rho(c))    ! output dry fluid threshold
+         wnd_frc_thr_slt = tmp2 / sqrt(forc_rho(c)) * frc_thr_wet_fct !* frc_thr_rgh_fct   ! fluid threshold
+         wnd_frc_thr_slt_it = B_it * tmp2 / sqrt(forc_rho(c)) ! define impact threshold
 
-            frc_thr_rgh_fct = 1.0_r8
+         ! the above formula is true for Iversen and White (1982) and Shao and Lu (2000) scheme
+         wnd_frc_thr(p) = wnd_frc_thr_slt          ! output fluid threshold
 
-            ! the following comes from subr. frc_thr_wet_fct_get
-            ! purpose: compute factor by which soil moisture increases threshold friction velocity
-            ! adjust threshold velocity for inhibition by moisture
-            ! modified 4/5/2002 (slevis) to use gravimetric instead of volumetric
-            ! water content
+         ! use emission threshold to calculate standardized threshold and dust emission coefficient
+         wnd_frc_thr_slt_std = wnd_frc_thr_slt * sqrt(forc_rho(c) / forc_rho_std) ! standardized soil threshold friction speed (defined using fluid threshold)
+         wnd_frc_thr_std(p) = wnd_frc_thr_slt_std          ! output standardized fluid threshold
+         dst_emiss_coeff(p) = Cd0 * exp(-Ce * (wnd_frc_thr_slt_std - wnd_frc_thr_slt_std_min) / wnd_frc_thr_slt_std_min) ! save dust emission coefficient here for all grids
 
-            bd = (1._r8-watsat(c,1))*2.7e3_r8      ![kg m-3] Bulk density of dry surface soil
-            gwc_sfc = h2osoi_vol(c,1)*SHR_CONST_RHOFW/bd    ![kg kg-1] Gravimetric H2O cont
-            if (gwc_sfc > gwc_thr(c)) then
-               frc_thr_wet_fct = sqrt(1.0_r8 + 1.21_r8 * (100.0_r8*(gwc_sfc - gwc_thr(c)))**0.68_r8)
+         ! framentation exponent
+         frag_expt = (Ca * (wnd_frc_thr_slt_std - wnd_frc_thr_slt_std_min) / wnd_frc_thr_slt_std_min)  ! fragmentation exponent, defined in Kok et al. (2014a)
+         if (frag_expt > 5.0_r8) then   ! set fragmentation exponent to be 3 or 5 at maximum, to avoid local AOD blowup
+            frag_expt = 5.0_r8
+         end if
+
+         !################ drag partition effect, and soil friction velocity ###########################
```

Comment:
@dmleung something that I thought that would help with clarity would be to spin off these sections that implement Leung_2023 on top of Kok_2013 is to turn these sections into subroutines: DragPartitionEffect, ImpactThreshold, CalcIntermittencyFactor.

This isn't required but it is something that could be done. I figure we should wait on it, but I wanted to get your input on names for the subroutines and how it would be broken up.

Reply:
In talking with @dmleung we thought that since splitting out Kok_2014 is not a priority we will probably leave it as is. It would be helpful to have the comments be very clear on what is the base Kok_2014 work, and what the Leung_2023 extension adds to it.